.github/workflows/mbsolr-tests.yml

```
@@ -0,0 +1,28 @@
+name: MusicBrainz Solr Packaging Tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Set up Eclipse Temurin 17
+        uses: actions/setup-java@v3
```

v4 is the latest, let's use that?

---

Yes, that’s because the file has been written in Nov 2022.

```suggestion
        uses: actions/setup-java@v4
```

---

Just pushed https://github.com/metabrainz/mb-solr/pull/55/commits/13bff381e1e41906600439478fd76d13bee72174